modules/http-api/src/main/java/org/mule/service/http/api/client/HttpClientConfiguration.java

```diff
@@ -84,6 +91,7 @@ public String getOwnerName() {
     private int maxConnections = -1;
     private boolean usePersistentConnections = true;
     private int connectionIdleTimeout = 30000;
+    private int responseBufferSize = 10240;
```

Comment:
mmm, can this use constants e.g. `DEFAULT_RESPONSE_BUFFER_SIZE` as defaults instead of literals, or are they not visible?